### PR TITLE
docs - add analyzedb best practice for partitioned tables

### PR DESCRIPTION
**Pull request by lisakowen**
analyze the root partition any time a new partition is added to a partitioned table.  also a few misc edits.

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml [hunk 1]
```diff
@@ -58,7 +58,14 @@
...
           statistics are collected for partitioned tables, see <codeph><xref
               href="../../ref_guide/sql_commands/ANALYZE.xml"
-        >ANALYZE</xref></codeph>.</p></sectiondiv>
+        >ANALYZE</xref></codeph>.</p>
+        <p><codeph>analyzedb</codeph> must sample additional partitions within a partitioned
+          table when it encounters a stale partition, even when statistics are already
+          collected. Consider it a best practice to run <codeph>analyzedb</codeph> on the
+          root partition any time that you add a new partition(s) to a partitioned table.
+          This operation both analyzes the child leaf partitions in parallel and merges
+          any updated statistices into the root partition.</p>
```
Comment (gpopt) [inline gpdb-doc/dita/utility_guide/ref/analyzedb.xml:67]: statistices  -> statistics

**Review by gpopt — APPROVED** [review]
+1

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml [hunk 2]
```diff
@@ -55,8 +55,16 @@ analyzedb { -? | -h | --help }</codeblock>
...
           statistics for non-current partitions and the root partition. For information about how
-          statistics are collected for partitioned tables, see <xref
-            href="../../ref_guide/sql_commands/ANALYZE.xml">ANALYZE</xref>.</p></sectiondiv>
+          statistics are collected for partitioned tables, see <codeph><xref
+              href="../../ref_guide/sql_commands/ANALYZE.xml"
+        >ANALYZE</xref></codeph>.</p>
+        <p><codeph>analyzedb</codeph> must sample additional partitions within a partitioned
+          table when it encounters a stale partition, even when statistics are already
+          collected. Consider it a best practice to run <codeph>analyzedb</codeph> on the
+          root partition any time that you add a new partition(s) to a partitioned table.
```
Comment (l-wang) [inline gpdb-doc/dita/utility_guide/ref/analyzedb.xml:64]: The term `root partition` always sounds weird to me. How could you be `root` and `partition` at the same time? But looks like it's already everywhere in our documentation and even in the syntax (`ANALYZE ROOTPARTITION...`). So I'm not really proposing to rename it in this PR. Upstream Postgres documentation usually says `root table`/`child table` or `partitioned table`/`leaf partitions`.

**Review by l-wang — APPROVED** [review]

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml (hunk 2 above)
Comment (gpopt) [reply gpdb-doc/dita/utility_guide/ref/analyzedb.xml:64]: Indeed.

Comment (lisakowen) [reply gpdb-doc/dita/utility_guide/ref/analyzedb.xml:64]: yes it confused me as well.  i attempted to use existing terminology in this PR.

**Pull request merged**